pandas/core/indexes/period.py

```diff
@@ -235,7 +235,16 @@ def _add_comparison_methods(cls):
 
     def __new__(cls, data=None, ordinal=None, freq=None, start=None, end=None,
                 periods=None, copy=False, name=None, tz=None, dtype=None,
-                **kwargs):
+                **fields):
```

Comment:
would be ok with making these explicit. cc @jbrockmendel

Reply:
+1

Reply:
It might be not very convenient, keeping in mind how it's used then in https://github.com/pandas-dev/pandas/blob/63ce781c9ebfb7d308adb3be62755e2bdf6c0d8e/pandas/core/indexes/period.py#L269-L270
I _can_ explicitly construct a dict from these arguments, but will it be better?